i_garantie_liste_implementieren

Sippl opened the pull request:
- Recycler View für Garantie Liste erstellt
- Klasse C_Garantie_Adapter hinzugefügt
- box_garantie_view hinzugefügt
- Datebankanbindung vorbereitet

Bitte daran denken, dass in C_Garantie_Adapter noch einiges zukünftig angepasst werden muss, sobald die Datenbankanbindung funktioniert. Vor allem der Konstruktor MyViewHolder ist momentan nur beispielsweise so eingebunden (mit Dummy Bild etc.).

Sippl committed:
i_garantie_liste by Fabi: XML für Recycler View erstellt, Recycler View in Klasse A_Garantie hinzugefügt, Adapter für Garantie hinzugefügt, Inital Test Werte von Tab_bons übernommen, noch nicht fertig & Bugfrei.

Sippl committed:
Layout Anpassung
- Title Bar dimen erstellt
- Dimen in die Garantie Content XML eingefügt

Sippl committed:
I_garantie_liste_implementieren:
- Anbindung Prepare Data für Datenbankzugriff angepasst.

Sippl committed:
I_garantie_liste_implementieren:
- onclick in xml rausgenommen

schmitzdominic committed:
Attribute gerefactort

Sippl committed:
I_garantie_liste_implementieren:
- assets rausgenommen

joeymez reviewed (changes requested):
Wenn ich nicht falsch liege erkenne ich beim schnellen durchschauen keinen (großen) Unterschied zum Code vom Marcus zu den Bons .... bis auf kleinere Änderung muss daher der Code korrekt sein, da Marcus sein Code fast 1:1 das selbe wiederspiegelt :)

- Was mir jedoch derzeit noch fehlt, sind die Methoden zum Deleten und Adden von Daten (C_Garantie_Adapter)
- Ich weiß das es nur Testdaten sind, dennoch könnte man die Logik bereits implementieren (würde später Zeit sparen :) ) ... konnte ich bei meiner Recyclerview für Budget auch machen 
- Wenn diese Implementiert wurden gibts auch von mir ein OK ^^

Sippl committed:
I_garantie_liste_implementieren:
- Implementation onClickListener für DeleteButton -> Noch nicht fertig!

Sippl committed:
I_garantie_liste_implementieren:
- Implementation onClickListener für DeleteButton -> Button reagiert und löscht Item jedoch random. Dieses Problem verschwindet sobald dies per Datenbank geregelt wird vermutlich von selbst.

joeymez reviewed (commented):
Alles klar ... unsere Änderungswünsche wurden berücksichtigt. Von meiner Seite wir das Item Freigegeben! :)

joeymez reviewed (approved):
Alles klar ... unsere Änderungswünsche wurden berücksichtigt. Von meiner Seite wir das Item Freigegeben! :)

schmitzdominic reviewed (approved):
Sieht sehr gut aus.
Funktioniert auch bei mir, sogar die Logs werden ausgegeben. Als kleinen Tipp, benutze Log.e dann siehst du das Kommentar auch weil es in Rot angezeigt wird ;)
